SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,397 @@
+Changes to the world are made by [workers](#worker). Each worker has a view onto the world (the part of the world that they're [interested](#interest) in), and SpatialOS sends them updates when anything changes in that view.
+It's important to recognise this fundamental separation between the SpatialOS world and the view of that world that a worker [checks out](#check-out). Workers send updates to SpatialOS when they want to change the world: they don't control the canonical state of the world, they must use SpatialOS APIs to change it.
+
+### Snapshot
+A snapshot is a representation of the state of a [SpatialOS world](#spatialos-world) at a given point in time. It stores each [persistent](#persistence) [SpatialOS entity](#spatialos-entity) and the values of their [SpatialOS components](#spatialos-component)' [properties (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#property).
+You'll use a snapshot as the starting point (using an an “initial snapshot”) for your [SpatialOS world](#spatialos-world) when you [deploy your game](#deployment).
+
+> Related:
+> 
+> * [How to generate a snapshot]({{urlRoot}}/content/generating-a-snapshot) (GDK documentation)
+> * [Snapshots (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/operate/snapshots)
+
+### Streaming queries
+Streaming queries allow workers to get information about the [world](#spatialos-world) outside the region they’re [interested in](#interest), so that they know about entities that they don’t have checked out (for example, entities that are far away, or that don’t have a physical position).
+
+> Streaming queries are useful if you need to get information about an entity periodically - for example, so that a player can see and interact with it.
+> If you just need information about an entity at one particular time, use [queries](#queries) instead.
+
+### Workers
+The [SpatialOS Runtime](#spatialos-runtime) manages the [SpatialOS world](#spatialos-world): it keeps track of all the [SpatialOS entities](#spatialos-entity) and their
+[SpatialOS components](#spatialos-component). But on its own, it doesn’t make any changes to the world.
+Workers are programs that connect to a SpatialOS world. They perform the computation associated with a world: they can read what’s happening, watch for changes, and make changes of their own.
+There are two types of workers; server-workers and client-workers. 
+A server-worker approximates to a server in native Unreal networking but unlike unreal networking, in SpatialOS, you have have more than one server-worker.
+
+* **Server-worker**<br/>
+    <br/>A server-worker is a [worker](#workers) whose lifecycle is managed by SpatialOS. When running a [deployment](#deployment), the SpatialOS Runtime starts and stops server-workers based on your chosen [load balancing (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#load-balancing) configuration.
```

Comment:
Not sure this enforced `<br/><br/>` break is needed here - we don't need a blank line between the bullet point and this para.